From 0f41600f3e2e38352f87b0a2d130e12b0da26ee3 Mon Sep 17 00:00:00 2001
From: Stefan Schantl <stefan.schantl@ipfire.org>
Date: Tue, 12 Feb 2013 20:26:41 +0100
Subject: [PATCH 16/17] Quality Agent: Replace "env ruby".

This commit adds a rule to replace "env ruby" with the correct run command "/usr/bin/ruby".
---
 tools/quality-agent.d/050-invalid-interpreters | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/quality-agent.d/050-invalid-interpreters b/tools/quality-agent.d/050-invalid-interpreters
index deac7a3..63847c7 100755
--- a/tools/quality-agent.d/050-invalid-interpreters
+++ b/tools/quality-agent.d/050-invalid-interpreters
@@ -28,7 +28,8 @@ check() {
 				sed -i ${file} \
 					-e "s,/usr/bin/env python\(.*\),/usr/bin/python\1," \
 					-e "s,/usr/bin/env python,/usr/bin/python," \
-					-e "s,/usr/bin/env perl,/usr/bin/perl,"
+					-e "s,/usr/bin/env perl,/usr/bin/perl," \
+					-e "s,/usr/bin/env ruby,/usr/bin/ruby,"
 
 				# If we could not fix it, raise an error
 				if [ "${interpreter}" = "$(file_get_interpreter ${file})" ]; then
-- 
1.8.1.2

